9-simple-exercises/state props/src/App.jsx

```diff
@@ -0,0 +1,18 @@
+import { useState } from "react";
+
+function App() {
+  const [ count, setCount ] = useState(0);
+
+  const clickHandler = () => setCount(prevCount => prevCount + 1);
+
+  return <div>
+    <div className="nav">
+      <a href="#">Home</a>
+      {[1, 2, 3, 4, 5, 6, 7, 8, 9].map(id => <a key={id} href="#">Exercise {id}</a>)}
+    </div>
+    <p>Button has clicked {count}</p>
+    <button onClick={clickHandler}>Click me</button>
+  </div>;
```

Comment:
You need create a button components and it is best to not use app while coding. just import your component here.